### ISSUE TITLE
Reduce excessive logging of LDAP app

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/core#17921]
Currently with a logging level of INFO, having a LDAP configured, if we don't set up a group filter, we are going to see loads of logs about it:

```
{"reqId":"P3pE0XNIp+HQ3Hb9cLXg","remoteAddr":"x.x.x.x","app":"user_ldap","message":"No group filter is specified, LDAP group feature will not be used.","level":1,"time":"2015-07-28T09:09:18+00:00"}
```

I think this info shouldn't appear all the time. It adds a lot of noise.

@blizzz

**Comment by blizzz** [comment]
Fixed with https://github.com/owncloud/core/issues/17921. 2 minute action even without bothering IDE.

**Comment by DeepDiver1975** [comment]
> Fixed with #17921. 2 minute action even without bothering IDE.

#17923
